core/os/memory.h

```diff
@@ -85,8 +85,19 @@ class Memory {
 	static uint64_t get_mem_available();
 	static uint64_t get_mem_usage();
 	static uint64_t get_mem_max_usage();
+
+	static constexpr size_t get_aligned_address(size_t p_address, size_t p_alignment);
 };
 
+constexpr size_t Memory::get_aligned_address(size_t p_address, size_t p_alignment) {
+	const size_t n_bytes_unaligned = p_address % p_alignment;
+	return (n_bytes_unaligned == 0) ? p_address : (p_address + p_alignment - n_bytes_unaligned);
+}
+
+constexpr size_t Memory::SIZE_OFFSET = 0;
+constexpr size_t Memory::ELEMENT_OFFSET = get_aligned_address(SIZE_OFFSET + sizeof(uint64_t), alignof(uint64_t));
+constexpr size_t Memory::DATA_OFFSET = get_aligned_address(ELEMENT_OFFSET + sizeof(uint64_t), alignof(max_align_t));
```

Comment:
This'll fix the CI errors
```suggestion
inline constexpr size_t Memory::SIZE_OFFSET = 0;
inline constexpr size_t Memory::ELEMENT_OFFSET = get_aligned_address(SIZE_OFFSET + sizeof(uint64_t), alignof(uint64_t));
inline constexpr size_t Memory::DATA_OFFSET = get_aligned_address(ELEMENT_OFFSET + sizeof(uint64_t), alignof(max_align_t));
```

Reply:
That's weird, `constexpr` implies `inline` according to my IDE.

Reply:
It implies it for functions, but not necessarily variables. It's strange that it's not implicit in a static class scope though